i18n/ja/docusaurus-plugin-content-docs/current/unraid-os/release-notes/7.2.2.md

```
@@ -8,12 +8,14 @@
 
 手順ごとの説明は、[Unraid のアップグレード](../../unraid-os/system-administration/maintain-and-update/upgrading-unraid.mdx) を参照してください。[ライセンス](../../unraid-os/troubleshooting/licensing-faq.mdx#license-types--features) に関する質問はこちらですか?
 
-### 既知の問題 {#known-issues}
+<span id="known-issues"></span>
 
+### 既知の問題
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)